src/api/users/test/e2e/utils.js

```diff
@@ -0,0 +1,140 @@
+const fetch = require('node-fetch');
+const { hash, createServiceToken } = require('@senecacdot/satellite');
+
+const USERS_URL = 'http://localhost/v1/users';
+
+// takes a user object, performs GET using the user's email
+const getUser = async (user) => {
+  const response = await fetch(`${USERS_URL}/${hash(user.email)}`, {
+    headers: {
+      Authorization: `bearer ${createServiceToken()}`,
```

Comment:
Should probably mention in the PR that Satellite needs to be updated for these tests to work. I had failed tests because the user service was on a lower version of Satellite